noop/unikernel.ml

```diff
@@ -0,0 +1,6 @@
+module Main = struct
```

Comment:
I am wondering wether it makes sense to make that unikernel even smaller by do not enclosing a `Main` module and having a single `start` value.